packages/dds/tree/src/feature-libraries/chunked-forest/codec/chunkDecodingGeneric.ts

```diff
@@ -57,6 +58,7 @@ export class DecoderContext<TEncodedShape = unknown> {
 		public readonly identifiers: readonly string[],
```

Comment:
I know its a preexisting issue, but since you added a parameter I noticed.

IIRC, the proper way to document parameter properties like these is with a doc comment on the parameter, not the constructor. That would mean moving the comment we have here down:

```suggestion
		/**
		 * Identifier substitution table (use to replace numeric identifier indexes with the actual identifiers from this table).
		 */
		public readonly identifiers: readonly string[],
```

Then following this paten to add a doc comment to the new paramater would be nice.